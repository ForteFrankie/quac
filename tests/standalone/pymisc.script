#!/bin/bash

# Test some Python stuff that needs a specific external environment.
#
# Copyright (c) Los Alamos National Security, LLC, and others.

. ./environment.sh
exec 2>&1

cd $DATADIR


echo '*** u.glob_maxnumeric()'

x touch a b
python3 -c 'import u; print(u.glob_maxnumeric("."))'
x touch 0
python3 -c 'import u; print(u.glob_maxnumeric("."))'
x touch 00
python3 -c 'import u; print(u.glob_maxnumeric("."))'
x touch 1 9
python3 -c 'import u; print(u.glob_maxnumeric("."))'

