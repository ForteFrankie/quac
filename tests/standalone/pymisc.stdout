*** u.glob_maxnumeric()
$ touch a b
None
$ touch 0
0
$ touch 00
0
$ touch 1 9
9
